File: solutions/0693-binary-number-with-alternating-bits/solution.py
class Solution(object):
    def hasAlternatingBits(self, n):
        """
        :type n: int
        :rtype: bool
        """
        count=n&1;
        n>>=1;
        while(n>0):
            if((n&1)==count):
                return False;
            count=(n&1);
            n>>=1;
        return True;   

File: solutions/0693-binary-number-with-alternating-bits/solution.js
/**
 * @param {number} n
 * @return {boolean}
 */
var hasAlternatingBits = function(n) {
    let count=n&1;
    n>>=1;
    while(n>0)
    {
        if((n&1)==count)
            return false;
        count=(n&1);
        n>>=1;
    }
    return true;
};
